### PR TITLE
Added Teams upcoming controls.

### PR DESCRIPTION
**Pull request by anchawl**
@levinec

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender/microsoft-secure-score-whats-coming.md [hunk 1]
```diff
@@ -30,11 +30,16 @@ Microsoft Secure Score can be found at https://security.microsoft.com/securescor
...
+### July 2021
 
-#### Remove improvement action related to Microsoft Cloud App Security
+#### Add improvement action related to Microsoft Teams
+
+- Restrict dial in users from bypassing a meeting lobby
+- Limit external participants to have control in a Teams Meeting
+- Restrict anonymous users from starting Teams meetings
+- Require lobbies to be set up for Teams Meetings
+- Configure which users are allowed to present in meetings
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/microsoft-secure-score-whats-coming.md]:
```suggestion
- Restrict dial-in users from bypassing a meeting lobby.
- Limit external participants from having control in a Teams meeting.
- Restrict anonymous users from starting Teams meetings.
- Require lobbies to be set up for Teams meetings.
- Configure which users are allowed to be present in Teams meetings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anchawl, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this Pr. Thanks

**Pull request merged**